tools/unit_testing/run_tests.py

```diff
@@ -289,7 +289,6 @@ def _main():
         os.environ["compile_threaded"] = "true"
     os.environ["CC"] = find_executable("mpicc")
     os.environ["FC"] = find_executable("mpif90")
-    os.environ["NETCDF_PATH"] = os.environ.get("NETCDF")
```

Comment:
I don't have the NETCDF environment variable defined on my Mac, so this line failed. Removing it allowed unit tests to build on my Mac, and didn't break them on yellowstone. However, if you feel this is needed in some cases, then we could do something like wrapping this in a conditional or try/except.